pkg/tsdb/tempo/trace_transform.go

```diff
@@ -210,11 +210,41 @@ func getAttributeVal(attr *commonv11.AnyValue) any {
 		return attr.GetBoolValue()
 	case *commonv11.AnyValue_DoubleValue:
 		return attr.GetDoubleValue()
-	case *commonv11.AnyValue_KvlistValue, *commonv11.AnyValue_ArrayValue:
-		return attr.GetStringValue()
+	case *commonv11.AnyValue_KvlistValue:
+		return kvListAsString(attr.GetKvlistValue())
+	case *commonv11.AnyValue_ArrayValue:
+		return arrayAsString(attr.GetArrayValue())
 	default:
-		return nil
+		return attr.GetStringValue()
+	}
+}
+
+func arrayAsString(list *commonv11.ArrayValue) string {
+	vals := make([]any, len(list.GetValues()))
+
+	for i, val := range list.GetValues() {
+		vals[i] = getAttributeVal(val)
+	}
+
+	res, err := json.Marshal(vals)
+	if err != nil {
+		return ""
```

Comment:
Can we log an error? Just so we know there is an issue, or for people to report when the values do not appear in the UI correctly. Something like we do for [tags](https://github.com/grafana/grafana/blob/main/pkg/tsdb/tempo/trace_transform.go#L144).

Also, if this returns an empty string, does that manifest itself in the UI?